install_scripts/templates/common/docker-install.sh

```diff
@@ -216,9 +215,11 @@ _installDocker() {
     compareDockerVersions "20.10.0" "${1}"
     if [ "$LSB_DIST" = "rhel" ] && [ "$COMPARE_DOCKER_VERSIONS_RESULT" -le "0" ]; then
         if ! yum list installed "yum-utils" >/dev/null 2>&1; then
+            logStep "Installing yum-utils"
             yum install -y -q yum-utils
+            logSuccess "Installed yum-utils"
         fi
-        yum-config-manager --add-repo=http://mirror.centos.org/centos/7/extras/x86_64
+        yum-config-manager --add-repo=http://mirror.centos.org/centos/7/extras/x86_64 || true
```

Comment:
could we use `DIST_VERSION_MAJOR` instead of `7` here?
```suggestion
        yum-config-manager --add-repo=http://mirror.centos.org/centos/$DIST_VERSION_MAJOR/extras/x86_64 || true
```

Reply:
(I just don't know whether using the centos 7 repos on centos/rhel/ol 8 is a good idea)